src/content/translations/pl/nft/index.md

```diff
@@ -0,0 +1,340 @@
+
+#### Minting NFTs {#minting-nfts}
+
+When you mint an NFT, a few things have to happen:
+
+- It needs to be confirmed as an asset on the blockchain.
+- The owner's account balance must be updated to include that asset. This makes it possible for it to then be traded or verifiably "owned".
+- The transactions that confirm the above need to be added to a block and "immortalized" on the chain.
+- The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximize their earnings.
+
+All these tasks are done by block producers and validators. Block proposers add your NFT transaction to a block and broadcast it to the rest of the network. Validators check that the transaction is valid and then add it to their databases. There are lots of crypto-economic incentives in place to make sure validators are acting honestly. Otherwise, anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership.
+
+#### NFT security {#nft-security}
+
+Ethereum's security comes from proof-of-stake. The system is designed to economically disincentivize malicious actions, making Ethereum tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ETH to change it. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT, and the bad actor would be economically penalized and ejected.
+
+Security issues relating to NFTs are most often related to phishing scams, smart contract vulnerabilities or user errors (such as inadvertently exposing private keys), making good wallet security critical for NFT owners.
+
+<ButtonLink to="/security/">
+  More on security
+</ButtonLink>
+
+## Build with NFTs {#build-with-nfts}
+
+Most NFTs are built using a consistent standard known as [ERC-721](/developers/docs/standards/tokens/erc-721/). However there are other standards that you might want to look into. The [ERC-1155](https://blog.enjincoin.io/erc-1155-the-crypto-item-standard-ac9cf1c5a226) standard allows for semi-fungible tokens which is particularly useful in the realm of gaming. And more recently, [EIP-2309](https://eips.ethereum.org/EIPS/eip-2309) has been proposed to make minting NFTs a lot more efficient. This standard lets you mint as many as you like in one transaction!
```

Comment:
Yeah, this whole section is still in English... do we know why?

Reply:
Bucket wasn't fully reviewed. It's more Polish than it was :P? Do you think we should scrap or keep it in this @wackerow ?